pkg/userinfo/roleRef_test.go

```diff
@@ -158,7 +158,7 @@ func Test_getRoleRefByRoleBindings(t *testing.T) {
 
 	list := make([]*rbacv1.RoleBinding, 2)
 
-	list[0] = newRoleBinding("test1", "mynamespace",
+	list[0] = newRoleBinding("test1", "default",
```

Comment:
`newRoleBinding` takes the namespace as the second para, we should update this function to actually take namespace from the para instead of hardcoding it.

Reply:
Okay @realshuting but from where does that para should get namespace from? From subjects? Since its a test, we can't get this para from user right?

Reply:
The namespace should be passed into `newRoleBinding`, but I don't see it is used in this function. What I was suggesting is to set the rolebinding's namespace from the namespace para.

Reply:
@mritunjaysharma394 - can you address comments?

Reply:
Yes @realshuting working on them